test/unit/index-test.js

```diff
@@ -496,11 +496,11 @@ describe('index.js', function() {
         it('instruments the inrepo addon', function() {
           const includes = Index._getIncludesForInRepoAddonDirectories();
           expect(includes).to.deep.equal([
-            'my-app/utils/my-covered-util.js',
```

Comment:
Do we no longer support folders for the app, in-repo-addon, etc? I think we should, right?

Reply:
have no idea(
This changes just to get tests passed

Reply:
Hmm, maybe something changed in the resolver or something. I don't think we want to change the paths though.

Reply:
how we can fix tests in this case?

Reply:
I tried to investigate this change and it seems like the test is green if i run `yarn node-test` locally in `code-coverage-upstream-oct-2019` with changes in `test/unit/index-test.js` reverted :thinking: 

It even fails if i keep the changed index-test.js 

![20191020-091030](https://user-images.githubusercontent.com/1205444/67156179-83a7f580-f31b-11e9-998b-df469f5f8063.png)

I tried this in both node8 and node latest

Reply:
I suppose it was failing on my windows machine, anyway changes reverted